src/components/ListMovies/ListMovies.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {render, userEvent} from '@testing-library/react-native';
+import List from './ListMovies';
+
+import {CardType} from '../../types/moviesTypes';
+import {mockListMovies} from '../../mock/MockData';
+
+type mockProps = {
+  data: CardType;
+};
+
+jest.mock('../MovieCard/MovieCard', () => {
+  const {Text} = require('react-native');
+  return {
+    MovieCard: ({data}: mockProps) => <Text>{data.Title}</Text>,
+  };
+});
+
+describe('renders ListMovies', () => {
+  it('should render ListMovies with an empty list', async () => {
+    const mockOnEndReached = jest.fn();
+    const {getByText} = render(
+      <List data={[]} onEndReached={mockOnEndReached} />,
+    );
+    getByText('Movies not found!');
+  });
+
+  it('should render ListMovies with movie titles', async () => {
+    const mockOnEndReached = jest.fn();
+    const {getByText} = render(
+      <List data={mockListMovies} onEndReached={mockOnEndReached} />,
+    );
+    mockListMovies.forEach(movie => {
+      expect(getByText(movie.Title)).toBeTruthy();
+    });
```

Comment:
I wouldn't really recommend to expect in cycles. Probably better approach would be this

```suggestion
    expect(mockListMovies.map(movie => !!queryByTestId(movie.Title))).toStricklyEqual([true, true, ...])
```